README.md

```diff
@@ -592,6 +592,15 @@ poetry2nix.mkPoetryApplication {
 }
 ```
 
+Please remember that `withDefaults` function, applies user overrides first, and than applies `defaults`. If you want to change package
```

Comment:
> Please remember that **the** `withDefaults` function

Missing the

> and **then**

Then instead of than.

> If you want to change **a** package

Missing a.